lang/python/python-typing-extensions/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2021 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-typing-extensions
+PKG_VERSION:=3.7.4.3
+PKG_RELEASE:=1
+
+PYPI_NAME:=typing_extensions
```

Comment:
```suggestion
PYPI_NAME:=typing-extensions
PYPI_SOURCE_NAME:=typing_extensions
```

Reply:
Fixed. Thanks for the suggestion. Next time I will use PYPI_SOURCE_NAME in a similar case.